docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md

```diff
@@ -192,42 +192,37 @@ You can create a `kubeconfig` file for previously created `yugabyte-platform-uni
 
 Continue configuring your Kubernetes provider as follows:
 
-- Give a meaningful name for your configuration.
-- Choose one of the folloiwng ways to specify **Kube Config** for an availability zone:
+- Specify a meaningful name for your configuration.
+- Choose one of the following ways to specify **Kube Config** for an availability zone:
   - Specify at **provider level** in the provider form. If specified, this configuration file is used for all availability zones in all regions.
   - Specify at **zone level** in the region form. This is required for **multi-az** or **multi-region** deployments.
-- Use **Service Account** to provide the name of the service account which has necessary access to manage the cluster (see [Create cluster](../../../../deploy/kubernetes/single-zone/oss/helm-chart/#create-cluster)).
-- Use **Image Registry** to specify from where to pull YugabyteDB image. Accept the default setting, unless you are hosting the registry.
-- Use the **Pull Secret File** field to upload the pull secret to download the image of the Enterprise YugabyteDB that is in a private repository. Your Yugabyte sales representative should have provided this secret.
-
-The following illustration shows the completed form:
-
-<img title="K8s Configuration -- filled" alt="K8s Configuration -- filled" class="expandable-image" src="/images/ee/k8s-setup/k8s-configure-filled.png" />
+- In the **Service Account** field, provide the name of the service account which has necessary access to manage the cluster (see [Create cluster](../../../../deploy/kubernetes/single-zone/oss/helm-chart/#create-cluster)).
+- In the **Image Registry** field, specify from where to pull the YugabyteDB image. Accept the default setting, unless you are hosting the registry.
+- Use **Pull Secret File** to upload the pull secret to download the image of the Enterprise YugabyteDB that is in a private repository. Your Yugabyte sales representative should have provided this secret.
 
 ## Configure region and zones
 
-Continue configuring your Kubernetes provider by clicking **Add Region** and completing the **Add new region** dialog, as follows:
+Continue configuring your Kubernetes provider by clicking **Add region** and completing the **Add new region** dialog, as follows:
+
+- Use the **Region** field to select the region.
 
-- Use the `Region` field to select the region.
 - Use the **Zone** field to select a zone label that should match with your failure domain zone label `failure-domain.beta.kubernetes.io/zone`.
+
 - Optionally, use the **Storage Class** field to enter a comma-delimited value. If you do not specify this value, it would default to standard. You need to ensure that this storage class exists in your Kubernetes cluster.
+
 - Use the **Namespace** field to specify the namespace. If provided SA has the `Cluster Admin` permissions, you are not required to complete this field. The SA used in the provided `kubeconfig` file should have access to this namespace.
```

Comment:
"SA" - service account? acronym not used previously

Reply:
Fixed